libs/qCC_db/ccPointCloud.cpp

```diff
@@ -431,12 +431,11 @@ void ccPointCloud::clear()
 
 void ccPointCloud::unalloactePoints()
 {
-	clearLOD();
```

Comment:
No we have to clear the LOD structure before clearing the colors / SFs. But we can't call 'notifyGeometryUpdate' before releasing them :D